README.md

```diff
@@ -90,7 +90,6 @@ Deploying using Helm consists of 4 steps
 
```

Comment:
(not about this line)
I spotted something odd about how this manual installation section starts.

![image](https://github.com/user-attachments/assets/ef8f99ef-e569-423a-9967-0077aa8394a2)

Could you please remove this first sentence from the line 82, if you agree?

> StackRox offers quick installation via Helm Charts.

Reply:
Done